tensorbay/opendataset/HardHatWorkers/loader.py

```diff
@@ -53,6 +53,10 @@ def _load_labels(label_file: str) -> List[LabeledBox2D]:
     labels = []
     for obj in label_tree.findall("object"):
         bndbox = obj.find("bndbox")
-        box = (int(child.text) for child in bndbox)  # type: ignore[arg-type, union-attr]
-        labels.append(LabeledBox2D(box, category=obj.find("name").text))  # type: ignore[union-attr]
+        labels.append(
+            LabeledBox2D(
+                *map(int, [child.text for child in bndbox]),  # type: ignore[arg-type, union-attr]
```

Comment:
`*(int(child.text) for child in bndbox)`
or
`*map(lambda child: int(child.text), bndbox)`
I prefer first one~